api/generator/generator.go

```diff
@@ -266,22 +266,15 @@ func (cg *CertGeneratorHandler) Handle(w http.ResponseWriter, r *http.Request) e
 			"certificate":         certSum,
 		},
 	}
+
+	if len(req.Request.Hosts) == 0 {
+		return api.SendResponseWithMessage(w, result, "This certificate lacks a \"hosts\" field. This makes it unsuitable for websites.", 5300)
```

Comment:
Instead of the numerical error, consider using the error code directly: New(PolicyError, InvalidRequest).ErrorCode